lib/seeds.rb

```diff
@@ -0,0 +1,9 @@
+require './app/models/message'
```

Comment:
I like the use of the seeds file to reduce the text used in the tests/development